index.html

```diff
@@ -56,7 +62,7 @@ <h3>Tonic</h3>
                 <p class="project_description">A daily selection of privately personalized reads; no 
                     accounts or sign-ups required.</p>
                 <footer>
-                    <ul class="skills">
+                    <ul class="skills" id="skills_1">
```

Comment:
- [ ] Kindly follow the Figma design because you must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). In skill portion your design is not same as Figma design please fix your skill badge design as mentioned in Figma Design check below  :point_down:  the screenshots for more clarification :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-02-23 22-05-40](https://user-images.githubusercontent.com/45226848/220987973-72f7f76c-4923-4bf1-9b07-733c129feb88.png)

**Figma Design** 
![Screenshot from 2023-02-23 22-04-08](https://user-images.githubusercontent.com/45226848/220987993-fc76e05d-0d26-47ee-9beb-43ae85c7ec24.png)

Reply:
Thank you for notifying and helping me to solve this. I have resolved the issue in the `skill `box as mentioned.